github/admin.go

```diff
@@ -62,6 +62,24 @@ func (m UserLDAPMapping) String() string {
 	return Stringify(m)
 }
 
+// Enterprise represents the GitHub enterprise profile
```

Comment:
In all the comments, please make complete sentences for the auto-generated godocs.
So please end each sentence with a period.